### PR TITLE
COMMERCE-6575 Skip portal instance

### PR DESCRIPTION
**Pull request by nikki-pru**
https://issues.liferay.com/browse/COMMERCE-6575

@iovaneale, This is separate from the Automation Tests from the interns. This is to fix the current functional tests that are failing at Setup with 'Search' errors.

cc/ @pierpinna

### CODE REVIEW

**Comment by nikki-pru** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
Thanks Nikki!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 1]
```diff
@@ -258,6 +259,7 @@ definition {
 
 	test CreateProductBundleWithPriceTypeDynamic {
 		property portal.acceptance = "true";
+		property skip.portal.instance = "CPCommerceProducts#CreateProductBundleWithPriceTypeDynamic";
```
Comment (iovaneale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:262]: Here, and for all the occurences, the right property is:

`property test.name.skip.portal.instance="Testcase#TestName";`

In this particular case there are two properties, just keep:

`property test.name.skip.portal.instance = "CreateProductBundleWithPriceTypeDynamic";`

**Review by iovaneale — CHANGES_REQUESTED** [review]

**Comment by nikki-pru** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
@iovaneale Fixed. Thanks

**Comment by nikki-pru** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
ci:forward

**Pull request closed without merging**